### PR TITLE
feat: all playlist pages

### PR DESCRIPTION
**Pull request by maitrakhatri**
# features added
- Liked Videos playlist page
- All Playlist pages
- Specific Playlist page
- User can remove videos from Liked and Specific Playlist page

### CODE REVIEW

**Review by Amandeep-sinha — APPROVED** [review]
Awesome UI 
 looks good to go ahead

File: src/Components/PlaylistIcon.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export function PlaylistIcon({ title, number }) {
+  return (
+    <div className="playlist">
+      <span class="name">
+        <img src="../assets/icons/playlist_play_black_36dp.svg" alt={title} />
+        {title}
+      </span>
+      <span class="no">{number}</span>
```
Comment (vaibhav18matere) [inline src/Components/PlaylistIcon.jsx:8]: make it `className`

**Review by vaibhav18matere — APPROVED** [review]
- good UI, informative naming
- delete icon should have better UI (red color) and exact at top right corner. it is below to that card component.
- card component on desktop screen looks much wider and less in height. (you can fix this later)
- or simply use different delete icon all together

File: src/Components/PlaylistIcon.jsx (hunk 1 above)
Comment (maitrakhatri) [reply src/Components/PlaylistIcon.jsx:8]: yes

**Comment by maitrakhatri** [comment]
> * good UI, informative naming
> * delete icon should have better UI (red color) and exact at top right corner. it is below to that card component.
> * card component on desktop screen looks much wider and less in height. (you can fix this later)
> * or simply use different delete icon all together

thanks for these, I'll implement them

**Pull request merged**